lib/molinillo/resolution.rb

```diff
@@ -461,11 +461,14 @@ def filter_possibilities_after_unwind(unwind_details)
       # @param [UnwindDetails] details of the conflict just unwound from
       # @return [void]
       def filter_possibilities_for_primary_unwind(unwind_details)
-        all_requirements = unwind_details.conflicting_requirements
```

Comment:
please keep this being extracted into a local

Reply:
Done (a couple of lines below).